app/experimenter/experiments/api/v5/types.py

```diff
@@ -81,6 +82,12 @@ class NimbusLabelValueType(graphene.ObjectType):
     value = graphene.String()
 
 
+class NimbusExperimentOwner(DjangoObjectType):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "first_name", "last_name", "email")
+
+
 class NimbusExperimentType(DjangoObjectType):
```

Comment:
You don't explicitly have to refer to it here?

Reply:
Nope, nor do we in the Nimbus models. I wanted to ensure they're the same model so I used the same function.